src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -3,46 +3,50 @@
 import java.util.List;
 import java.util.Objects;
 
-public class LottoNumber {
-    public static final int MINIMUM_CANDIDATE_NUMBER = 1;
-    public static final int MAXIMUM_CANDIDATE_NUMBER = 45;
-    public static final String NUMBER_OUT_OF_BOUNDS_ERROR = String.format(
-            "%d 에서 %d 사이의 숫자만 입력해주세요.",
-            MINIMUM_CANDIDATE_NUMBER,
-            MAXIMUM_CANDIDATE_NUMBER
-    );
-
-    private final int number;
-
-    public LottoNumber(int number) {
-        validateRangeOfNumber(number);
-        this.number = number;
-    }
-
-    private void validateRangeOfNumber(int number) {
-        if (number < MINIMUM_CANDIDATE_NUMBER || MAXIMUM_CANDIDATE_NUMBER < number) {
-            throw new IllegalArgumentException(NUMBER_OUT_OF_BOUNDS_ERROR);
-        }
-    }
-
-    public boolean hasAnyMatchingNumber(List<LottoNumber> numbers) {
-        return numbers.contains(new LottoNumber(number));
-    }
-
-    public String getNumberAsString() {
-        return String.valueOf(number);
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        LottoNumber that = (LottoNumber) o;
-        return number == that.number;
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(number);
-    }
+public class LottoNumber implements Comparable<LottoNumber> {
+	public static final int MINIMUM_CANDIDATE_NUMBER = 1;
+	public static final int MAXIMUM_CANDIDATE_NUMBER = 45;
+	public static final String NUMBER_OUT_OF_BOUNDS_ERROR = String.format(
+			"%d 에서 %d 사이의 숫자만 입력해주세요.",
+			MINIMUM_CANDIDATE_NUMBER,
+			MAXIMUM_CANDIDATE_NUMBER
+	);
+	private final int number;
+
+	public LottoNumber(int number) {
+		validateRangeOfNumber(number);
+		this.number = number;
+	}
+
+	@Override
+	public int compareTo(LottoNumber that) {
+		return this.number - that.number;
+	}
+
+	private void validateRangeOfNumber(int number) {
+		if (number < MINIMUM_CANDIDATE_NUMBER || MAXIMUM_CANDIDATE_NUMBER < number) {
+			throw new IllegalArgumentException(NUMBER_OUT_OF_BOUNDS_ERROR);
+		}
+	}
+
+	public boolean isIncludedIn(List<LottoNumber> numbers) {
+		return numbers.contains(new LottoNumber(number));
```

Comment:
```suggestion
		return numbers.contains(this);
```
eq & hc 도 재정의를 했으니 요렇게 하면 되겠네요 :)